content/blog/tech/arc-max-quand-l-IA-simplifie-notre-navigation-sur-le-web.md

```diff
@@ -0,0 +1,98 @@
+---
+
+Comme vous le savez, chez Elao ainsi que dans d’autres structures, l’IA prend de plus en plus de place que ce soit à travers nos projets internes ou pour le compte de nos clients.
+
+À l’agence, nous sommes plusieurs à utiliser quotidiennement le [navigateur ARC](https://arc.net/). Basé sur Chromium (également utilisé par Google Chrome), nous avons été ravis d’apprendre que de nouvelles fonctionnalités basées sur l’intelligence artificielle avaient vu le jour.
+
+On vous en dit plus !
+
+## ARC by The Browser Company
+
+Sorti il y a un peu plus d’un an (le 19 avril 2022) par la société The Browser Company, ce navigateur a su faire parler de lui grâce à son interface novatrice.
+
+The Browser Company part du postulat que l’ensemble des logiciels auxquels nous sommes confrontés quotidiennement tendent à évoluer, à proposer des fonctionnalités plus performantes et fluides. Or, les navigateurs eux, restent inchangés.
+
+C’est pourquoi ils ont souhaité penser le navigateur comme un véritable outil qui gagnerait à être amélioré afin de permettre un gain d’efficacité pour les utilisateurs. L’idée : réorganiser le navigateur traditionnel pour optimiser son temps de travail, bonifier sa créativité et ses capacités.
+
+## ARC ou le nouveau navigateur révolutionnaire
+
+Les points centraux de ce navigateur : la navigation, la personnalisation et l’organisation.
+Oubliez l’utilisation des onglets traditionnels et leur accumulation ! ARC se propose de naviguer… verticalement ! Doté d’une sidebar, tout se passe à ce niveau-là.
+
+Avec ARC, vous pouvez :
+- Créer des espaces de travail et y retrouver les liens que vous utilisez le plus souvent. Par exemple, si vous avez une casquette communicationnelle en plus d’une casquette opérationnelle, vous pouvez vous créer deux espaces de travail distincts dans lesquels vous retrouverez les liens, les logiciels qui sont propres à vos deux activités.
+- Créer plusieurs dossiers dans ces espaces de travail.
+
+![Organisation des espaces de travail](content/images/blog/2023/arc-max-quand-l-ia-simplifie-notre-navigation-sur-le-web/organisation-espace-travail.png)
+
+- Garder en mémoire vos onglets épinglés que vous pourrez retrouver même après avoir fermé le navigateur.
+- Choisir un mode focus en faisant disparaître la sidebar.  
+- Dupliquer votre écran afin d’avoir deux pages différentes sur la même interface.
+
+![Duplication de l'écran](content/images/blog/2023/arc-max-quand-l-ia-simplifie-notre-navigation-sur-le-web/duplication-ecran.png)
+
+Le tout repose sur une interface fluide et plaisante. Bien qu’il faille un petit temps d’adaptation pour se familiariser avec le principe de la sidebar qui remplace les onglets que l’on a l’habitude d’utiliser, on s'accommode facilement à la souplesse de ce navigateur.
+Petit bémol cependant, ARC est disponible uniquement sous MacOS (pour le moment) !
+
+Si vous souhaitez en savoir plus sur toutes les fonctionnalités de ce navigateur, c’est par ici.
```

Comment:
[**Question**] : `c'est par ici` signifie l'article courant ? ou c'est censé être un lien vers une autre ressource ? 

```diff
- c'est par ici.
+ vous êtes au bon endroit.
```

J'ai l'impression que ma formulation est moins équivoque, mais c'est sujet à interprétation, bien entendu ! Ne pas hésiter à ignorer mon commentaire si je suis le seul à me poser la question.

Reply:
@xavier-rdo c'est vers un lien sur la vidéo de The Browser Company, qui explique les nouvelles fonctionnalités, d'ailleurs @quentin-brouillet si on peut rajouter le lien : https://www.youtube.com/watch?v=y7koAGLf0EE